### PR TITLE
[Y_Demkiv] Task 2

### PR DESCRIPTION
**Pull request by YarynaD**
https://yarynad.github.io/ui-internship-lwo2/css-practice/task_2/index.html

### CODE REVIEW

File: css-practice/task_2/index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Exo" rel="stylesheet">
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: add this link above own styles

File: css-practice/task_2/index.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    <title>Document</title>
+</head>
+<body>
+    <div class="container"> 
+        <header>
+            <h1>There is no other platforms for you as like </h1>
+            <form class="user-actions" action="#">
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: why you use `<form>` tag here ?

File: css-practice/task_2/index.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                        </p>
+                    </div>
+                </article>
+                <div class="empty"></div> 
+            </div>
+            <section class="about">
+                <img class="penholder" src="img/Pen_holder.png" alt="pen holder">
+                <img class="earphones" src="img/Earphones.png" alt="earphones">
+                <img class="coffe" src="img/kisspng-coffee-cup-black-coffee-cup-5aa22cf303b3e8.9451227715205777790152.png" alt="cofee_cup">
+                <h3>NEW FEATURES</h3>
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: use text-transform

File: css-practice/task_2/index.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+                <article>
+                    <h3>NEW DESIGN</h3>
+                    <h2>Resposive design, just need your tap <span>....</span></h2>
+                    <p>Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiasem malesuada magn. Vestibulum felis euismod semper.</p>
+                </article>
+            </div>
+            <div class="partnership"> 
+                    <div class="co-founders">
+                        <div class="founder">
+                            <div></div>
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: why you use empty block?

File: css-practice/task_2/index.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+                            <h5>Jonathon Doe</h5>
+                            <p class='position'>Co Founder</p>
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: use double quotes in html

Comment (Nazar-bnm) [reply css-practice/task_2/index.html]: Check everywhere

File: css-practice/task_2/index.html [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+                        </div>
+                        <div class="founder">
+                            <div></div>
+                                <h5>Jonathon Doe</h5>
+                                <p class='position'>Co Founder</p>
+                                <p>“Aenean eu leo quam. Pellentesque ornare sem lacinia qua emere wancerid elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiasem malesuada magn”</p>
+                        </div>
+                    </div>
+                    <div class="partners">
+                        <img src="img/Logo1.png" alt="logo">
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: more informative name for files

File: css-practice/task_2/index.html [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+            </div>
+            <p>By Signing up you agree to our <a href="#">terms & Services.</a></p>
+        </aside>
+        <footer>
+            <div class="footer-wrapper">
+                <div class="adress">
+                    <h5>Start<span>Ex</span></h5>
+                    <img src="img/Polygon_2.png" alt="arrow">
+                    <p>Aenean lacinia bibendum nulla sed consectetur. Aenean eu leo quam. 
+                            USA &  CAN: 1-888-123-4567
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: regular HTML character codes for `&`

File: css-practice/task_2/index.html [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+                    <h5>About US</h5>
+                    <a href="#">About Us</a>
+                    <a href="#">Terms of Use</a>
+                    <a href="#">Privacy Policy</a>
+                </div>
+                <div class="contact-us">
+                    <h5>Get Newsletter</h5>
+                    <div class="e-mail">
+                        <input type="text" placeholder="EMAIL">
+                        <button></button>
```
Comment (Nazar-bnm) [inline css-practice/task_2/index.html]: empty button

File: css-practice/task_2/sass/_media-small.scss [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+    }
+    .intro {
+        article {
+        width: 100%;
+        h2 {
+            font-size: 30px;
+        }
+        }   
+    }
+    .penholder, .earphones, .coffe {
```
Comment (Nazar-bnm) [inline css-practice/task_2/sass/_media-small.scss]: coffee

File: css-practice/task_2/sass/_media-small.scss [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+            padding-left: 0;
+            input:first-child {
+                margin-left: 29px;
+            }
+        }
+        .pointer {
+            width: 260px;
+            display: inline-flex;
+            height: 43px;
+            @include align-self(flex-end);
```
Comment (Nazar-bnm) [inline css-practice/task_2/sass/_media-small.scss]: move to top

File: css-practice/task_2/sass/_media-small.scss [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+            margin-top: 7px;
+            margin-bottom: 16;
```
Comment (Nazar-bnm) [inline css-practice/task_2/sass/_media-small.scss]: what are the units ?

File: css-practice/task_2/sass/_media-small.scss [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+            margin-bottom: 6px;
```
Comment (Nazar-bnm) [inline css-practice/task_2/sass/_media-small.scss]: duplication

File: css-practice/task_2/sass/_media-x-small.scss [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+            margin-left: 0;
+            input:first-child {
+                margin-left: 29px;
+            }
+        }
+        .pointer {
+            width: 260px;
+            display: inline-flex;
+            height: 43px;
+            @include align-self(flex-end);
```
Comment (Nazar-bnm) [inline css-practice/task_2/sass/_media-x-small.scss]: move to top

File: css-practice/task_2/sass/_media-x-small.scss [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+            margin-top: 7px;
+            margin-bottom: 16;
+            margin-bottom: 6px;
```
Comment (Nazar-bnm) [inline css-practice/task_2/sass/_media-x-small.scss]: duplication

File: css-practice/task_2/sass/_media768-1023.scss [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+        .feedback {
+            width: 79%;
+            display: inline-block;
+            text-align: center;
+            padding-left: 10%;
+            input:first-child {
+                margin-left: 29px;
+            }
+        }
+        .pointer {
```
Comment (Nazar-bnm) [inline css-practice/task_2/sass/_media768-1023.scss]: duplication styles

File: css-practice/task_2/index.html (hunk 1 above)
Comment (YarynaD) [reply css-practice/task_2/index.html]: I tried both google fonts links and they are perfect for Exo Regular and Light from template. But not for Exo Thin which is in the header of template. That's why I dowloaded it separately and used @font-face.

File: css-practice/task_2/index.html (hunk 2 above)
Comment (YarynaD) [reply css-practice/task_2/index.html]: I read in some tutorial that buttons and inputs should wraped inside <form> tag. Is it wrong?

File: css-practice/task_2/index.html (hunk 4 above)
Comment (YarynaD) [reply css-practice/task_2/index.html]: I use empty div to upload image as it's background. It is the founders photo before information about him. Should I change this div to img?

File: css-practice/task_2/index.html (hunk 8 above)
Comment (YarynaD) [reply css-practice/task_2/index.html]: It's empty, because it has no text - just an image on it. I upload image using background.

**Pull request merged**